app/src/org/commcare/activities/CommCareSetupActivity.java

```diff
@@ -384,6 +387,12 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
                     result = data.getStringExtra(InstallArchiveActivity.ARCHIVE_JR_REFERENCE);
                 }
                 break;
+            case GET_APPS_FROM_HQ:

```

Comment:
There is lots of error messaging for if the request fails in the `InstallFromListActivity` itself. If they've gotten to this point, it either means that the request succeeded and the InstallFromListActivity was automatically finished after they selected 1 of the apps, or they manually pressed back without selecting an app, and thus we don't need/want to do anything here.